nexus/src/app/background/init.rs

```diff
@@ -180,8 +180,8 @@ impl BackgroundTasks {
                 String::from("external_endpoints"),
                 String::from(
                     "reads config for silos and TLS certificates to determine \
-                    the right set of HTTP endpoints, their HTTP server names, \
-                    and which TLS certificates to use on each one",
+                     the right set of HTTP endpoints, their HTTP server \
+                     names, and which TLS certificates to use on each one",
```

Comment:
it looks like Rustfmt wants subsequent lines of multi-line strings to be indented by an additional space so that the first character of each line lines up vertically (compensating for the opening `"`)? 

i don't have a problem with this in the abstract, but it's something we'll have to remember to do manually until the string-wrapping support stabilizes... do you know if there's a way to change this behavior? it's not a huge deal if not.

Reply:
[I don't see a way to customize this behavior](https://rust-lang.github.io/rustfmt/?version=master&search=format_strings#format_strings).

It is unfortunate that "format_strings" remains unstable but it's been at least four years so I assume it's not stabilizing soon.  In the meantime I assume we're just making our best manual effort at consistency.  The 80-column issue is much more impactful to me than the leading space, and there are quite a lot of long strings in Omicron, and I've considered fixing them all in one PR, but it would only be a one-time fix.  I'd welcome better approaches here!

Reply:
:woman_shrugging: fine with me, then!